File: src/Gang/GangCommandWrapper.cs
namespace Gang
{
    public sealed class GangCommandWrapper : IGangCommandWrapper
    {
        public GangCommandWrapper(
            string type,
            object command)
        {
            Type = type;
            Command = command;
        }

        public string Type { get; }
        public object Command { get; }
    }
}

File: src/GangJs/Core/docs/media/models/communications/GangCommandWrapper.js
/** wraps command data */
export class GangCommandWrapper {
    constructor(
    /** command data type */
    type, 
    /** command data */
    data, 
    /** sequence number */
    sn = undefined, 
    /** in reply to sequence number */
    rsn = undefined) {
        this.type = type;
        this.data = data;
        this.sn = sn;
        this.rsn = rsn;
    }
}
